syslogs.txt

```diff
@@ -58,6 +58,7 @@
 %ASA-1-114001: Failed to initialize 4GE SSM I/O card (error error_string).
 %ASA-1-114002: Failed to initialize SFP in 4GE SSM I/O card (error error_string).
 %ASA-1-114003: Failed to run cached commands in 4GE SSM I/O card (error error_string).
+%ASA-3-114013: Failed to set mac address table in 4GE SSM I/O card (error error_string).
```

Comment:
Sorry I did not see this change during office hours. You shouldn't have needed to add this line - it should have already been down lower in the file - here is a link to that [line](https://github.com/NCATComp410/comp410_summer21/blob/a4381241ca0e425091395496e87aafec01c1922d/syslogs.txt#L265)

Reply:
Should I delete the line I added and then merge my request?

Reply:
Best thing to do is delete that line on your local machine then re-run your unit-test to make sure it still works, then commit and push and your pull request should update automatically

Reply:
Hey professor I updated the pull request but it says I need to ask for review again, could you do it, or do I need to ask my teammates again?